os/board/stm32l4r9ai-disco/src/stm32_exflash.c

```diff
@@ -125,8 +125,10 @@ void stm32_exflash_initialize(void)
 
 #if defined(CONFIG_FLASH_PARTITION)
     int ret;
+	struct mtd_dev_s *mtd;
```

Comment:
I know your alignment is correct, but there are many different alignments. Please fix them or matched to them.

Reply:
Modified